python/setup.cfg

```diff
@@ -17,14 +17,12 @@ universal = 1
 [flake8]
 exclude = docs
 select = ANN,B,B9,B950,BLK,C,D,E,F,I,S,W
-ignore = E203,E510,W503,S101,D412,W503
-extend-ignore = E203
+ignore = ANN101,ANN102,S101,D412,W503
 max-line-length = 140
 max-complexity = 10
 application-import-names = pynessie,tests
 import-order-style = smarkets
 docstring-convention = google
-per-file-ignores = tests/*:S101
```

Comment:
note that `ignore`, `extend-ignore` and `per-file-ignores` all built on top of eachother... so since `S101` and `E203` are in `ignore` already they were not needed in the later stages

also removed the `Exx` ones, since they are no longer present in the codebase.